lms/djangoapps/ccx/views.py

```diff
@@ -7,6 +7,7 @@
 import json
 import logging
 import pytz
+from math import ceil
```

Comment:
nit: Looking at the organization of the code, this should go after line 12 (in the `from` section)